catalogue/webapp/components/WorkDetailsProperty/WorkDetailsProperty.tsx

```diff
@@ -1,11 +1,13 @@
-// @flow
-import { type Node } from 'react';
 import { font } from '@weco/common/utils/classnames';
 import SpacingComponent from '@weco/common/views/components/SpacingComponent/SpacingComponent';
+import { FunctionComponent, ReactNode } from 'react';
 
-type Props = {| title: ?string, children: Node |};
+type Props = { title?: string; children: ReactNode };
 
-const WorkDetailsProperty = ({ title, children }: Props) => {
+const WorkDetailsProperty: FunctionComponent<Props> = ({
```

Comment:
You can also use the `PropsWithChildren` type.

Reply:
If you work this out I'd be grateful - you [shouldn't](https://www.carlrippon.com/react-children-with-typescript/) actually need anything other than `FunctionComponent<Props>` as that does the injection of children itself (have a look in its definition), but doing that and removing the non-injected `: Props` after the destructure didn't work - could it be a linting rule or a package version problem?

Reply:
I've had the same issue. It's not a type issue, but rather [a linting rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md) issue. 

I personally prefer using the `FunctionComponent` type to infer what the props are and could be, so if we can't find a way to make the linter happy I might have a look at turning the rule off as I'd say static typing trumps linting.

Reply:
I'm going to get this in i nit's current state, and we can discuss as a team later.